tests/www/presenters/test_show_company_work_invite_details_presenter.py

```diff
@@ -28,7 +28,7 @@ def test_successful_response_renders_to_view_model(self) -> None:
     def test_successful_response_renders_accept_url_to_view_model(self) -> None:
```

Comment:
Tests the web presenter has the properly parameterized `get_answer_company_work_invite_url` passed.